tracer/src/Datadog.Trace.BenchmarkDotNet/DatadogDiagnoser.cs

```diff
@@ -70,8 +110,235 @@ public void Handle(HostSignal signal, DiagnoserActionParameters parameters)
     /// <inheritdoc />
     public void DisplayResults(ILogger logger)
     {
+        if (_enableProfiler)
+        {
+            if (_platformNotSupportedException is null)
+            {
+                logger.WriteLine($"Datadog Profiler was attached.");
+            }
+            else
+            {
+                logger.WriteLine("Datadog Profiler: " + _platformNotSupportedException.Message);
+            }
+        }
     }
 
     /// <inheritdoc />
     public IEnumerable<ValidationError> Validate(ValidationParameters validationParameters) => Array.Empty<ValidationError>();
+
+    private static IEnumerable<string> GetHomeFolderPaths(DiagnoserActionParameters parameters)
+    {
+        // try to locate it from the environment variable
+        yield return EnvironmentHelpers.GetEnvironmentVariable("DD_DOTNET_TRACER_HOME");
+
+        // try to locate it inside the running benchmark folder
+        yield return Path.Combine(parameters.Process.StartInfo.WorkingDirectory, "datadog");
+
+        // try to locate it inside the benchmark folder
+        yield return Path.Combine(Path.GetDirectoryName(parameters.BenchmarkCase.Descriptor.Type.Assembly.Location) ?? string.Empty, "datadog");
+
+        // try to locate it in the default path using relative path from the benchmark assembly.
+        yield return Path.Combine(
+            Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) ?? string.Empty,
+            "..",
+            "..",
+            "..",
+            "..",
+            "..",
+            "..",
+            "..",
+            "shared",
+            "bin",
+            "monitoring-home");
```

Comment:
I wrapped this with a preprocessor constant just because it's easier for debugging, to include that we have to build the project with: `dotnet build -c Release -p:ExtraDefineConstants=LOCAL_HOME_FOLDER`

Reply:
If it's easier, I'd be fine with `#if DEBUG` too 🙂